src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/PublicationCreateRequest.cs

```diff
@@ -24,4 +25,14 @@ public string Slug
     }
 
     public Guid? SupersededById { get; set; }
+
+     public class Validator : AbstractValidator<PublicationCreateRequest>
+    {
+        public Validator()
+        {
+            RuleFor(request => request.Title)
+                .NotEmpty()
+                .MaximumLength(65);
+        }
+    }
```

Comment:
Just spotted that we are also validating summary is less than 160 chars so we can move this to the Fluent Validation validator now too to be consistent.

We can also write unit tests for this.

Reply:
Fixed in 4701db8.